command.go

```diff
@@ -216,6 +227,17 @@ func cmdTailscaleProxy(fs caddycmd.Flags) (int, error) {
 		},
 		AppsRaw: appsRaw,
 	}
+	if debug {
```

Comment:
Question: is the caddy config generated here persisted across restarts? If it is, then users would have no way to unset the debug flag except for by deleting the config

Reply:
no, this config is entirely in memory.  It's not persisted anywhere.

In hindsight, I'm not actually sure how useful this `tailscale-proxy` subcommand really is compared to just creating a pretty simple caddyfile that does the same thing.  It was something I added pretty on, mostly just to see if I could 🤷🏻

Reply:
would the proxy subcommand be useful for the Dockerfile? https://github.com/tailscale/caddy-tailscale/issues/14

Reply:
yeah, possibly.  If the Dockerfile just provided a caddy binary with the tailscale plugin compiled in, then you could use that to run the proxy.  I'll go ahead and add that to the v1 milestone... seems worth doing.